test/controllers/BusinessVerificationControllerSpec.scala

```diff
@@ -810,7 +813,7 @@ class BusinessVerificationControllerSpec extends PlaySpec with GuiceOneServerPer
     val userId = s"user-${UUID.randomUUID}"
 
     AuthBuilder.mockAuthorisedUser(userId, mockAuthConnector)
-    when(mockBackLinkCache.fetchAndGetBackLink(ArgumentMatchers.any())(ArgumentMatchers.any())).thenReturn(Future.successful(None))
+    when(mockBackLinkCache.fetchAndGetBackLink(ArgumentMatchers.any())(ArgumentMatchers.any())).thenReturn(Future.successful(Some("/ated-subscription/appoint-agent")))
```

Comment:
This comes from the ated-subscription-frontend as a query parameter (right?). Should we retain an AT for this in that case?

Reply:
I have retained an AT for this